frontend/app/src/store/status.ts

```diff
@@ -87,7 +87,7 @@ export const useStatusStore = defineStore('status', () => {
   const shouldShowLoadingScreen = (section: Section, subsection: string = defaultSection) => computed<boolean>(() => {
     const statuses = get(status)[section];
     if (!statuses)
-      return false;
+      return true;
```

Comment:
this one should be true, because if it's not found, it means the status is None, hence it should be true